.circleci/config.yml

```diff
@@ -4,6 +4,29 @@
 #
 version: 2
 jobs:
+  remix-libs:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:10.18.0-browsers
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+    resource_class: xlarge
+      # - image: circleci/mongo:3.4.4
+    environment:
+      - COMMIT_AUTHOR_EMAIL: "yann@ethereum.org"
+      - COMMIT_AUTHOR: "Circle CI"
+    working_directory: ~/remix-project
+
+    parallelism: 20
```

Comment:
This step is not needed since we are not making use of parallelism for running tests.